### DISCUSSION TITLE
Export/ Import Data

### DISCUSSION THREAD

**Discussion by mdev64** [NginxProxyManager/nginx-proxy-manager#4124] [Ideas]
Today, I lost all my data while trying to update the version. It was a mistake not to create a backup. Can we add an option to export and import all the data? This feature could be very useful for anyone who needs to migrate the server.

**Comment by larz99** [comment]
Yes Please!  I haven't lost my configuration YET, but this is an important core feature.
Many Thanks in advance,
Larz

**Comment by akaii78** [comment]
Same here...
I had to reconfigure all my hosts...

**Comment by sector7gp** [comment]
I dont think this feature demands a lot of code, a simple json export/import to the current DB engine will be fine!

**Comment by pricard32** [comment]
+1 I really want a Import/Export function in the setting page.

+2 if we could schedule an auto-backup and send it on cloud/samba

**Comment by markmigliorini** [comment]
Me too
I dont think this feature demands a lot of code, a simple json export/import to the current DB engine will be fine!
